src/web/DomElement.C

```diff
@@ -1123,7 +1123,7 @@ std::string DomElement::createVar() const
 {
 #ifndef WT_TARGET_JAVA
   char buf[20];
-  std::sprintf(buf, "j%d", nextId_++);
+  std::sprintf(buf, "j%u", nextId_++);
```

Comment:
...and this one?

This would use `std::memory_order_seq_cst`, which theoretically has more overhead. I experimented a bit and the difference seems to be negligible in practice. On AMD64 it would generate the same assembly code, on AArch64 performance was very close. So I would be fine with just all of them using `operator++` instead of `fetch_add`.

Reply:
No objections against using `operator++` on my part. This actually makes switching between the atomic and the not-atomic version much easier.